plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/StandardProjectWizard.java

```diff
@@ -20,12 +39,18 @@
 
 public class StandardProjectWizard extends Wizard implements INewWizard {
 
-  private AppEngineStandardWizardPage page;
+  private WizardPage page;
   private AppEngineStandardProjectConfig config = new AppEngineStandardProjectConfig();
+  private boolean hasAppEngineComponent;
 
   public StandardProjectWizard() {
     this.setWindowTitle("New App Engine Standard Project");
-    page = new AppEngineStandardWizardPage();
+    hasAppEngineComponent = doesAppEngineJavaComponentExist();
+    if (hasAppEngineComponent) {
```

Comment:
I think it would be more helpful if this page did the following:
  - check if there is a Cloud SDK: if none (or not found), then show a `Button` or `Link` that would open the Cloud SDK pref page to configure the location.
  - if missing java components: provide a `Button` or `Link` that would offer to invoke `gcloud components install app-engine-java` with a little transcript to show the results of the command.  If successful, mark the page as valid so the user can move on with _Next_.